test/binomial.jl

```diff
@@ -0,0 +1,28 @@
+using StatsFuns
+using Base.Test
+
+#Test Compares the difference in values w.r.t Rmath implementation
+
+N = Int64.( (0, 1, 1, 8, 20, 20, 20, 150, 700, 900, 1000) );
+P = (0.0, 0.5, 0.7, 0.6, 0.34, 0.89, 0.53, 0.77, 0.98, 0.5, 0.29);
+
+@testset "binompdf" begin
+for i = 1:length(N)
+    for x = Int64(0):N[i]

```

Comment:
Isn’t this inferred to be `Int64` by default?

Reply:
Yep, 2 year old PR, back when I had no idea how Julia worked, will fix this tomorrow.